src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/ClientAnalyzerBase.cs

```diff
@@ -91,7 +91,6 @@ protected static IMethodSymbol FindMethod(IEnumerable<IMethodSymbol> methodSymbo
 
             return methodSymbols.SingleOrDefault(symbol =>
             {
-
```

Comment:
If there are no changes to this file, it would be better to revert it so it doesn't appear in the list of changed files.  This makes it easier to understand how the repo has changed in the future.